index.html

```diff
@@ -4,35 +4,80 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css"/>
     <title>YouTube Embedded Video Page</title>
   </head>
   <body>
-    <header class="home-search">
-      <img src="index.jpg" alt="YouTube image" />
-      <input
-        type="text"
-        name="search bar"
-        title="search bar"
-        placeholder="Search"
-      />
-      <button type="submit"><i class="fas fa-search"></i></button>
+    <header id="nav-bar">
+      <img id="logo" src="index.jpg" alt="YouTube image" />
+      <div id="searchView">
+        <input
+          type="text"
+          name="search bar"
+          title="search bar"
+          placeholder="Search"
+        />
+        <button type="submit"><i class="fas fa-search"></i></button>
+      </div>
+      <div id="rightNavView">
+        <button id="searchButton" type="submit"><i class="fas fa-bars"></i></button>
+        <button id="searchButton" type="submit"><i class="fas fa-ellipsis-v"></i></button>
+        <button id="searchButton" type="submit"><i class="fas fa-upload"></i></button>
+      </div>
     </header>
-    <section class="position">
-      <iframe
-        width="635"
-        height="397"
-        src="https://www.youtube.com/embed/V74l_zS1x8E"
-        frameborder="0"
-        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-        allowfullscreen
-      ></iframe>
-    </section>
-    <aside class="position">
-      <div id="sidebar">
-        <img src="" alt="image1" />
-        <img src="" alt="image2" />
-        <img src="" alt="image3" />
+    <section id="content">
```

Comment:
You could use the `<main>` HTML tag instead of the `<section>` for the code to be more semantic